http4s-server-micrometer/src/main/scala/com/avast/sst/http4s/server/micrometer/MicrometerHttp4sMetricsOpsModule.scala

```diff
@@ -0,0 +1,50 @@
+package com.avast.sst.http4s.server.micrometer
+
+import java.util.concurrent.TimeUnit
+
+import cats.effect.Sync
+import cats.syntax.flatMap._
+import cats.syntax.functor._
+import io.micrometer.core.instrument.MeterRegistry
+import org.http4s.metrics.{MetricsOps, TerminationType}
+import org.http4s.{Method, Status}
+
+object MicrometerHttp4sMetricsOpsModule {
```

Comment:
I'm wondering whether we need this `*Module` suffix everywhere :thinking: 
but thankfully this is not the most burning question

Reply:
![image](https://user-images.githubusercontent.com/9019485/66558760-68d3c500-eb54-11e9-849f-dfa7e5ee173a.png)